src/components/UI/Text/InputNumber.vue

```diff
@@ -173,7 +173,13 @@ export default {
     },
     isValidationEnabled() {
       // whether input validation is on
-      return this.validation["enabled"] && this.validationMessage != "";
+      if(this.localValue.toString() == "0")
+      {
+        return this.validationMessage != "";
+      }
+      else{
+        return this.validation["enabled"] && this.validationMessage != "";
+      }
```

Comment:
this will still lead to the issue that I can add as many trailing zeros as I want and the eventual phone number length goes way beyond 10 and it is still marked valid (as shown in the image - no error message)

<img width="1022" alt="Screenshot 2022-01-31 at 10 18 00 AM" src="https://user-images.githubusercontent.com/13185699/151740436-db9a09a2-c32b-43e5-85ac-9d868e6283a7.png">